db/migrate.sh

```diff
@@ -6,7 +6,7 @@ export AWS_PROFILE=identity
 # flyway/flyway:11-alpine
 FLYWAY_CONTAINER="flyway/flyway@sha256:1850b2e6b257f774cdd6ad7554dc53cc278351ff0202f7b9b696ceafccbea493"
 
-Grey=$"\033[30m"
+Magenta=$"\033[35m"
```

Comment:
Also fix the grey text on gray background issue by switching to Magenta 
<img width="784" alt="image" src="https://github.com/user-attachments/assets/162e9c2c-fee0-4a5c-8d12-5ad90ac789a8" />